wandb/sdk/launch/_launch_add.py

```diff
@@ -156,7 +154,7 @@ def _launch_add(
     priority: Optional[int] = None,
 ) -> "public.QueuedRun":
     launch_spec = construct_launch_spec(
-        uri,
+        None,
```

Comment:
Why not remove the URI from the launch spec construction as well?

Reply:
There is some stuff there having to do with placeholder uris for sweep runs? or sweep schedulers?